### PR TITLE
update deps

### PR DESCRIPTION
**Pull request by hirosassa**
SSIA, for security reason on pillow.

### CODE REVIEW

**Comment by hirosassa** [comment]
The following change will cause error on the redis lock tests.
I'll be fixed this problem on the other PR.

https://github.com/dsoftwareinc/fakeredis-py/compare/v1.7.4...v1.7.5#diff-288db5468d2e9d706d7767762175c8b7f73e00a2601114e0be0c42b43d8f2843R2802-R2809

**Comment by hirosassa** [comment]
@Hi-king @mski-iksm @snowhork @kitagry @ujiuji1259 plz review.

File: poetry.lock [hunk 1]
```diff
@@ -538,23 +544,27 @@ test = ["hypothesis (>=3.58)", "pytest (>=6.0)", "pytest-xdist"]
 
 [[package]]
 name = "pillow"
-version = "9.0.1"
+version = "9.2.0"
```
Comment (hirosassa) [inline poetry.lock:547]: This is the main purpose of this PR.

**Review by Hi-king — APPROVED** [review]
LGTM

We need another changes to use recent fakeredis :)

**Comment by hirosassa** [comment]
Yes, we need to catch up latest fakeredis and redis, too.

**Review by kitagry — APPROVED** [review]

**Pull request merged**